pages/ato.md

```diff
@@ -26,6 +26,8 @@ To start the security clearance process, [create an issue in the DevOps reposito
     * <url>
     * ...
 * **Site:** <url>
+* **Product manager:** @username
+* **Technical lead:** @username
```

Comment:
Why don't we call it 'system owner' since that's the nomenclature used on
the SSP and other ATO docs?

On Thu, Dec 17, 2015 at 9:37 AM, Aidan Feldman <notifications@github.com>
wrote:

> In pages/ato.md
> <https://github.com/18F/before-you-ship/pull/32#discussion_r47911791>:
>
> > @@ -26,6 +26,8 @@ To start the security clearance process, [create an issue in the DevOps reposito
> >      * <url>
> >      * ...
> >  * **Site:** <url>
> > +* **Product manager:** @username
> > +* **Technical lead:** @username
>
> @micahsaul <https://github.com/micahsaul> @theresaanna
> <https://github.com/theresaanna> @kaitlin <https://github.com/kaitlin>
> @jgrevich <https://github.com/jgrevich> @sashax
> <https://github.com/sashax> We had decided (this summer?) to not create a
> formal Technical Lead position for each project, though acknowledged a
> primary use case would be ATOs. Would it make sense to leave this in here?
> Maybe it's just "technical lead" in the scope of the ATO?
>
> /cc @hollyallen <https://github.com/hollyallen>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/18F/before-you-ship/pull/32/files#r47911791>.
>



-- 
Kaitlin Devine
18F

Need to schedule a meeting? Check out my Calendly page
<https://calendly.com/kaitlin-devine>